app/controllers/activities_controller.rb

```diff
@@ -4,6 +4,25 @@
 class ActivitiesController < AuthenticatedController
   include ProjectScoped
 
+  def index
+    activities = Activity.includes(:trackable)
```

Comment:
As in L28 activities should be loaded through `current_project`.

Reply:
Thanks, fixed!